index.html

```diff
@@ -194,48 +197,52 @@ <h3 class="work-title">My Work Title</h3>
 							<li>Javascript</li>
 						</ul>
 					</div>
-					<button class="see-project-btn">See Project</button>
+					<button class="see-project-btn font-inter">See Project</button>
 				</div>
 			</div>
 		</section>
 		<section class="about-me" id="about-me">
 			<div class="resume">
-				<h1>About me</h1>
-				<p class="resume-text">
+				<h1 class="font-crete">About me</h1>
```

Comment:
- [ ] Kindly add some spacing between the `about me` section and the white horizontal line in `desktop-version` to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/167266325-0647f2da-3e91-40bf-a2f3-573cdae0d105.png) | ![image](https://user-images.githubusercontent.com/24703171/167266416-b63476fa-9978-4a16-8ac7-8801c19bae80.png) |